### PR TITLE
Only restore and remove if backup exists

### PR DESCRIPTION
**Pull request by maghuro**
To prevent `rm` and exec errors
Also Uncomment clear on menu

### CODE REVIEW

**Comment by dave14305** [comment]
The restore/delete menu choices are only displayed if the file is present, so it isn’t required to check again before running the command.

**Comment by maghuro** [comment]
I know, but if you press the number even if they aren't displayed, it throws the errors I mentioned. So it's better to check if user press 5 even if 5 isn't being displayed

**Comment by maghuro** [comment]
https://gofile.io/d/QOaUz7

**Comment by maghuro** [comment]
@dave14305 
added "no backup" message if user forces choosing option 5 or 6.

File: flexqos.sh [hunk 1]
```diff
@@ -801,7 +801,7 @@ prompt_restart() {
 } # prompt_restart
 
 menu() {
-	# clear
+	clear
```
Comment (maghuro) [inline flexqos.sh:804]: not sure if you want it to 'clear', but if not, please merge and comment clear later.

**Pull request merged**